katello_certs_tools/katello_ssl_tool.py

```diff
@@ -931,18 +974,19 @@ def genServerRpm(d, verbosity=0):
     fo.write('%s.src.rpm\n' % os.path.basename(serverRpmName))
     fo.write('%s\n' % os.path.basename(tarballFilepath))
     fo.close()
-    os.chmod(latest_txt, 0600)
+    os.chmod(latest_txt, '0600')
```

Comment:
`os.chmod` does not accept string parameters (at least not in Python 3.6 and 3.7)